install_config/install/prerequisites.adoc

```diff
@@ -214,14 +214,71 @@ The following must be set up in your environment before {product-title} can be
 installed.
 
 [[prereq-dns]]
-
 === DNS
 
+A fully functional DNS environment is an absolute requirement for
+{product-title} to work correctly. It is not enough to simply add entries for
```

Comment:
@tpoitras I don't think the first sentence is necessary, but if you want to keep it, I'd get rid of the "absolute". Also, second sentence is expletive. Maybe:

"Adding entries into the *_/etc/hosts_* file is not enough, because..."